psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -820,10 +820,10 @@ void OEProp::compute() {
     }
     // print_header();  // Not by default, happens too often -CDS
     if (tasks_.count("ESP_AT_NUCLEI")) compute_esp_at_nuclei();
-    if (tasks_.count("DIPOLE")) compute_dipole(false);
-    if (tasks_.count("QUADRUPOLE")) compute_quadrupole(false);
-    if (tasks_.count("TRANSITION_DIPOLE")) compute_dipole(true);
-    if (tasks_.count("TRANSITION_QUADRUPOLE")) compute_quadrupole(true);
+    if (tasks_.count("DIPOLE")) compute_multipoles(1, false);
+    if (tasks_.count("QUADRUPOLE")) compute_multipoles(2, false);
+    if (tasks_.count("TRANSITION_DIPOLE")) compute_multipoles(1, true);
+    if (tasks_.count("TRANSITION_QUADRUPOLE")) compute_multipoles(2, true);
```

Comment:
```suggestion
    if (tasks_.count("QUADRUPOLE")) compute_multipoles(2, false);
    else if (tasks_.count("DIPOLE")) compute_multipoles(1, false);
    if (tasks_.count("TRANSITION_QUADRUPOLE")) compute_multipoles(2, true);
    else if (tasks_.count("TRANSITION_DIPOLE")) compute_multipoles(1, true);
```
The `compute_multipoles` performs all orders up to the requested order, right? In that case, may want to consolidate tasks to avoid duplicate printing.